Minor patches to samples

shubham7109 opened the pull request:
This PR addresses minor issues with various samples. Changes made are described below:

- Display map: Updated screenshot
- Configure subnetwork trace: Added scroll view to work in portrait and landscape mode. 
- Create and save map: Changed viewpoint scale to show USA extent
- Open mobile map package: Improved starting position to show off buildings better
- List KML contents: Added text `Root KML node` to the first level blank node

TADraeseke reviewed (changes requested):
Thanks @shubham7109! Just a couple things in line below.

shubham7109 commented on kotlin/configure-subnetwork-trace/src/main/res/layout/activity_main.xml:
```diff
@@ -1,202 +1,210 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```
Fixed margins alignments and balanced spinners. There were a couple other layout style issues which are fixed. The UI elements should now feel in place rather than floating around oddly.

shubham7109 commented on java/open-mobile-map-package/src/main/java/com/esri/arcgisruntime/sample/openmobilemappackage/MainActivity.java:
```diff
@@ -53,6 +53,7 @@ protected void onCreate(Bundle savedInstanceState) {
       if (mMapPackage.getLoadStatus() == LoadStatus.LOADED && !mMapPackage.getMaps().isEmpty()) {
         // add the map from the mobile map package to the MapView
         mMapView.setMap(mMapPackage.getMaps().get(0));
+        mMapView.setViewpointScaleAsync(1000000);
```
Added a custom camera point at `mSceneView.setViewpointCamera(new Camera(39.962551,-75.177168,313.862355,138.654067,74.581665,0.0));`

shubham7109 commented:
- Added a custom camera viewpoint to `Open Mobile Scene Package`
- Fixed odd UI layout style for `Configure Network Trace`. Balanced margins, aligned spinners, made spinner items more visible and increased text size for visibility. 

Ready for review @TADraeseke

TADraeseke commented on kotlin/configure-subnetwork-trace/src/main/res/layout/activity_main.xml:
```diff
@@ -6,175 +6,213 @@
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/barrierConditionsTextView" />
+
```
whitespace

TADraeseke commented on kotlin/configure-subnetwork-trace/src/main/res/layout/activity_main.xml:
```diff
@@ -6,175 +6,213 @@
+
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
```
whitespace

TADraeseke commented on kotlin/configure-subnetwork-trace/src/main/res/layout/activity_main.xml:
```diff
@@ -199,4 +237,5 @@
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent" />
 
+
```
whitespace

TADraeseke commented on kotlin/configure-subnetwork-trace/src/main/res/layout/activity_main.xml:
```diff
@@ -6,175 +6,213 @@
+                    android:layout_marginTop="8dp"
+                    android:textOff="@string/falseText"
+                    android:textOn="@string/trueText"
+                    android:visibility="visible"
```
setting this to gone will keep the button from popping up during onCreate, then disappearing

TADraeseke reviewed (changes requested):
Thanks @shubham7109 -- Just a couple small things below. Thanks!

TADraeseke reviewed (approved):
LGTM! Thanks @shubham7109!
